src/Sulu/Bundle/MediaBundle/Media/FormatManager/DefaultFormatManager.php

```diff
@@ -191,21 +191,23 @@ public function returnImage($id, $formatName)
                 }
             } catch (MediaException $e) {
                 // return when available a file extension icon
-                list($image, $status, $imageExtension) = $this->returnFileExtensionIcon($formatName, $this->getRealFileExtension($fileName));
+                list($responseContent, $status, $imageExtension) = $this->returnFileExtensionIcon($formatName, $this->getRealFileExtension($fileName), $e);
             }
```

Comment:
do not nest try/catch blocks with the same Exception type ... this makes it really difficult to read ...